extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -0,0 +1,260 @@
+          "type": "integer",
+          "description": "The campaign activity originating this message."
+        },
+        "https://ns.adobe.com/experience/campaign/delivery/channel": {
+          "title": "Communication Channel",
+          "type": "string",
+          "meta:enum": {
+            "https://ns.adobe.com/xdm/channels/email": "Email",
+            "https://ns.adobe.com/xdm/channels/sms": "Mobile (SMS)",
+            "https://ns.adobe.com/xdm/channels/phone": "Phone",
+            "https://ns.adobe.com/xdm/channels/direct-mail": "Direct Mail",
+            "https://ns.adobe.com/xdm/channels/fax": "Fax",
+            "https://ns.adobe.com/xdm/channels/apns":
+              "Apple Push Notification Service",
+            "https://ns.adobe.com/xdm/channels/gcm": "Google Cloud Messaging",
+            "agency": "Agency",
+            "other": "Other"
+          },
+          "description": "The marketing channel used to execute this activity."
+        },
+        "https://ns.adobe.com/experience/campaign/campaign/id": {
+          "title": "CampaignID",
+          "type": "integer",
+          "description":
+            "The marketing campaign to which activity originating this message belongs to.."
+        },
+        "https://ns.adobe.com/experience/campaign/delivery/from": {
+          "title": "From",
+          "type": "string",
+          "description":
+            "Address used as a from/sender/caller address. Depending on the communication channel, the `address` has a different format.\n\n  * `email`: an email address.\n  * `textMessage` or `phone`: a shortcode or phone number."
+        },
+        "https://ns.adobe.com/experience/campaign/delivery/testEnabled": {
+          "title": "Test Enabled",
+          "type": "boolean",
+          "description":
+            "Specify whether campaign activity is of type A/B testing.",
+          "default": false
+        },
+        "https://ns.adobe.com/experience/campaign/delivery/messagesClass": {
+          "title": "Message Class",
+          "type": "string",
+          "meta:enum": {
+            "oneTime": "Marketing (One Time)",
+            "continuous": "Recurring",
+            "messageCenter": "Transactional (Event-based)"
+          },
+          "description":
+            "Marketing classification of the messages sent through that delivery.\n",
+          "default": 0
+        },
+        "https://ns.adobe.com/experience/campaign/delivery/templateID": {
+          "title": "Template",
+          "type": "integer",
+          "description":
+            "The delivery template's ID used to initialize this delivery.\n\nThe type the template used in Adobe Campaign can be identified using the `messageClass`.\n\n* `messageClass` = `oneTime`: the template is standard delivery template.\n* `messageClass` = `continuous`: the template is a recurring delivery.\n* `messageClass` = `transactional`: the template is a transactionnal message template.\n"
```

Comment:
The description is inconsistent with the keys defined above.

Reply:
Not sure, but I changed the title to "Identifier of the Template" if that's what you mean.

Reply:
You had different keys in the description than you defined in `meta:enum` above.

Reply:
In particular: `messagesClass` above, `messageClass` here.